webapp/components/survey/NodeDefDetails/store/hooks/useBasicProps.js

```diff
@@ -0,0 +1,60 @@
+import * as R from 'ramda'
+
+import * as Survey from '@core/survey/survey'
+import * as NodeDef from '@core/survey/nodeDef'
+import * as NodeDefLayout from '@core/survey/nodeDefLayout'
+
+import { useSurvey, useSurveyCycleKey } from '@webapp/store/survey'
+
+import * as NodeDefState from '../state'
+
+export const useBasicProps = (props) => {
+  const { nodeDefState } = props
+
+  const survey = useSurvey()
+  const surveyCycleKey = useSurveyCycleKey()
+
+  const nodeDef = NodeDefState.getNodeDef(nodeDefState)
+  const validation = NodeDefState.getValidation(nodeDefState)
+
+  const isEntityAndNotRoot = NodeDef.isEntity(nodeDef) && !NodeDef.isRoot(nodeDef)
+  const displayAsEnabled = isEntityAndNotRoot
+  const displayInEnabled = isEntityAndNotRoot
+  const displayAsFormDisabled = false
+  const displayAsTableDisabled = Survey.hasNodeDefChildrenEntities(nodeDef)(survey) || NodeDef.isSingle(nodeDef)
+
+  const displayInParentPageDisabled = NodeDefLayout.isRenderForm(surveyCycleKey)(nodeDef)
+  const displayInOwnPageDisabled = false
```

Comment:
maybe this variable and its use is not needed

Reply:
uhm, yes, same story as above.